formats/json-tests/commonTest/src/kotlinx/serialization/features/EmojiTest.kt

```diff
@@ -6,13 +6,15 @@ package kotlinx.serialization.features
 
 import kotlinx.serialization.builtins.serializer
 import kotlinx.serialization.json.JsonTestBase
+import kotlinx.serialization.test.*
 import kotlin.test.Test
 
 
 class EmojiTest : JsonTestBase() {
 
     @Test
     fun testEmojiString() {
+        if (isWasm()) return
```

Comment:
Is there an issue with TODO for this test?

Reply:
Please take a look

Reply:
This seems to be a bug in okio. I keep this test ingored for wasm and put the [link](https://github.com/square/okio/issues/1333) to the issue nearby.

Reply:
Great, thank you!

Reply:
https://youtrack.jetbrains.com/issue/KT-60212